mri/numerics/reconstruct.py

```diff
@@ -96,6 +99,16 @@ def sparse_rec_fista(gradient_op, linear_op, prox_op, cost_op,
         print(" - alpha variable shape: ", alpha.shape)
         print("-" * 40)
 
+    # Define the proximity dual operator
+    weights = copy.deepcopy(alpha)
+    weights[...] = mu
```

Comment:
The mu variable is not defined.
Please remove it

Reply:
Yes, I think thats why the Travis-build is failing. 
Also, if your updates affect the tests, can you please update them and ensure they are sane?

Reply:
Actually it's just because this part of the code is not wanted anymore, right?
We don't set the threshold within that method but actually, have it set before in the operator, right?

Reply:
Exactly, if you really need the weights take them from the proximity operator